style.css

```diff
@@ -115,26 +118,69 @@ html, body {
 .header__link {
     text-transform: uppercase;
     font-size: 12px;
+    letter-spacing: 1px;
     text-decoration: none;
-    color: #000000;
+    color: var(--main-text-color);
     font-family: 'Montserrat', sans-serif;
-    font-weight: bold;
+    cursor: pointer;
+}
+.header__link:hover, .header__cartlink:hover, .header__sublink:hover {
+    color: #a0522d;
 }
 .header__cartlink {
     font-size: 12px;
-    text-decoration: none;
-    color: #000000;
+    text-decoration: none;  
+    color: var(--main-text-color);
     font-family: 'Poppins', sans-serif;
     font-weight: bold;
     text-align: center;
- }
+}
+.form-search {
+    display: block;
+}
+.form-search input {
+    all: unset;
+    font-size: 10px;
+    letter-spacing: 2px;
+    width: 80%;
+}
+.form-search input[type="image"] {
+    width: 15px;
+    height: 15px;
+}
+.header__subnav {
+    display: none;
+}
+.header__subnav.active {
+    display: block;  
+    position: absolute;
+    left: -20px;
+    z-index: 2;
+    padding: 10px 10px 10px;
+    width: 110px;
+    height: 125px;
+    background-color: var(--main-bg-color);  
+}
+.header__sublink {
+    font-size: 12px;
+    letter-spacing: 1px;
+    text-decoration: none;  
+    color: var(--main-text-color);
+    font-family: 'Poppins', sans-serif;
```

Comment:
Potentially you could create a variable for the fonts that you keep reusing as well.

Reply:
Ok, changed fonts to var like colors  (don't know right it or not)